testing/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains an integration test that confirms the basic functionality
+of the profiler works on Compute Engine. In particular, this test confirms that 
+the agent can create and upload profiles from a Comput Engine VM, and that
+these profiles contain symbolized samples from the benchmark application.
+
+More specifically, this test:
+1. Starts 3 Compute Engine VMs, one to test Node.js versions 6, 8, and 9. 
```

Comment:
Nit: indent the second+ lines of a bullet formatting to align with the text. E.g. instead of
```
1. Starts 3 Compute Engine VMs, one to test Node.js versions 6, 8, and 9. 
Each Compute Engine VM then:
```
do
```
1. Starts 3 Compute Engine VMs, one to test Node.js versions 6, 8, and 9. 
   Each Compute Engine VM then:
```

Reply:
Done.